### ISSUE TITLE
Fullscreen

### ISSUE THREAD

**Issue by fgrs** [NUKnightLab/StoryMapJS#285]
does it actually support fullscreen?

**Comment by JoeGermuska** [comment]
No, at this time there is no support for toggling a StoryMap into "full screen" presentation.

It's not out of the question, but it isn't on the roadmap at this time.
